### DISCUSSION TITLE
Have created a Vercel integration, need help with login into server

### DISCUSSION THREAD

**Discussion by Abhay07** [vercel/community#964] [Help]
Hi Community,
As the title suggests, Have created a Vercel integration, got the Access keys, but need help with login into Vercel integration server. Have gone through the official docs.

**Comment by amyegan** [comment]
Thanks for reaching out @Abhay07. I'm not sure I understand exactly what you're trying to do. Can you share the doc or steps that you followed so I can give it a try? 

I think you might be trying to [use an access token with the API](https://vercel.com/docs/integrations/vercel-api-overview#), or maybe you want to [submit an integration to be officially listed](https://vercel.com/docs/integrations/submit-integration). But I don't want to make assumptions and give you bad information.

**Reply by Abhay07** [reply]
Ok. I created a vercel integration. Filled all the required fields. Got the Client Id, Client Secret. Want to know how to login to the server.

**Reply by amyegan** [reply]
The documentation on [creating an integration](https://vercel.com/docs/integrations/create-integration) has all of the steps you should need for making a new integration. I'm not sure what server you're referring to. Can you share a link to the step that you are unable to complete?
